### ISSUE TITLE
Replace `exec` to `spawn` for running tested app

### ISSUE THREAD

**Issue by miherlosev** [DevExpress/testcafe#2857]
When a tested app command (provided via the `--app` parameter) outputs a large number of log messages to `stdout` or `stderr`, it can overflow child process buffers used by the `child_process.exec` function. We should use `child_process.spawn` that doesn't have this limitation.

**Comment by AndreyBelym** [comment]
Workarounds:
##### macOS/Linux
```
testcafe --app "yarn start >/dev/null 2>&1" 
```

##### Windows
```
testcafe --app "yarn start >NUL"
```

**Comment by AndreyBelym** [comment]
Closed by f163ccb.
